client/components/jetpack/card/jetpack-product-card/index.tsx

```diff
@@ -115,6 +115,15 @@ const JetpackProductCard: React.FC< OwnProps > = ( {
 		  } )
 		: null;
 
+	const starIcon = (
```

Comment:
Would it be possible to leave a comment here stating that the inline SVG code is helpful because Safari has trouble rendering SVGs inside `<img>` tags correctly on Retina displays? I wouldn't have been able to tell the purpose of this PR, had I not looked into the Slack conversation you cited in the description.

Reply:
In the end it's up to you 😉

My reasoning for including a brief comment in code -- even if it's just a shorthand reference to the initial discussion or the URL of this pull request -- is because I can definitely see someone coming in at a later date and saying "Gee, I could clean this up by extracting the SVG into its own file and referencing it inside an `<img>` tag" which would undo the fix you just made 😅

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/65304/commits/08510114c3fff831b05ef67a5cfe6f9b32e56504

Reply:
Awesome, thanks! 🏁 Approved.